lib/sms_autofill.dart

```diff
@@ -70,8 +70,9 @@ class PinFieldAutoFill extends StatefulWidget {
       this.inputFormatters,
       this.enableInteractiveSelection = true,
       this.controller,
-      this.decoration = const UnderlineDecoration(
-          colorBuilder: FixedColorBuilder(Colors.black), textStyle: TextStyle(color: Colors.black)),
+      this.decoration,
+      //  = const UnderlineDecoration(
```

Comment:
Wonder if it would work up here, but simply removing the "const" in front.

Reply:
i was thinking that we keep it nullable, and use null operator to set the default value. made the other changes.